docs/guide-disappearance.md

```diff
@@ -3,93 +3,136 @@ id: guide-disappearance
-when the element is removed.
+When you want to wait until an element has disappeared, you can use
+`waitForElementToBeRemoved` or `waitFor`.
+
+The `waitForElementToBeRemoved` [async wait utility][async-api] takes an element
+as an argument and waits until that element has been removed from the document.
+
+```javascript
+test('movie title no longer present in document', async () => {
+  // element is present
+  const movie = screen.queryByText('the mummy')
+  await waitForElementToBeRemoved(movie)
 
-```jsx
-test('movie title no longer present in DOM', async () => {
-  // element is removed
-  await waitForElementToBeRemoved(() => queryByText('the mummy'))
+  // element has been removed
+  expect(movie).not.toBeInTheDocument()
 })
 ```
 
-Using
+`waitForElementToBeRemoved` works by using a
 [`MutationObserver`](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver)
-is more efficient than polling the DOM at regular intervals with `waitFor`.
+internally, which means it responds to the element being removed in a very
+efficient way.
+
+You can also pass an array or a callback to `waitForElementToBeRemoved` -
+[see the documentation for more options](dom-testing-library/api-async.md#waitforelementtoberemoved).
+
+Another option is using the `waitFor` helper.
 
-The `waitFor` [async helper][async-api] function retries until the wrapped function
-stops throwing an error. This can be used to assert that an element disappears
-from the page.
+The `waitFor` [async wait utility][async-api] retries until the wrapped function
+stops throwing an error. Because expectations throw errors when they fail,
+putting one in the wrapped function can be used to wait for the element to be
+removed.
 
-```jsx
+```javascript
 test('movie title goes away', async () => {
   // element is initially present...
   // note use of queryBy instead of getBy to return null
   // instead of throwing in the query itself
   await waitFor(() => {
-    expect(queryByText('i, robot')).not.toBeInTheDocument()
+    expect(screen.queryByText('i, robot')).not.toBeInTheDocument()
   })
 })
 ```
 
+Polling like this is not as efficient as observing for mutations using
+`waitForElementToBeRemoved`, but sometimes it's the best option.
+
 ## Asserting elements are not present
 
-The standard `getBy` methods throw an error when they can't find an element, so
-if you want to make an assertion that an element is _not_ present in the DOM,
-you can use `queryBy` APIs instead:
+As opposed to waiting for removal, this is for when you are at a point in your
+test where you know an element shouldn't be present.
+
+You might reach for `getBy` to check that something is not present, but `getBy`
+queries throw an error when they can't find an element, which means you don't
+get the chance to use the result in an expectation. If you want to make an
+assertion that an element is _not_ present in the document, you can use
+`queryBy` queries instead:
```

Comment:
I'd use more strong language here:

```suggestion
You might reach for `getBy` to check that something is not present, but `getBy`
queries throw an error when they can't find an element. If you want to make an
assertion that an element is _not_ present in the document, you need to use
`queryBy` queries instead:
```

Reply:
I'm not sure I understand how removing that clause results in stronger language, but I'm fine with the change.

Reply:
Oh, the change was from "you can use" to "you need to". Then I noticed the clause was kinda redundant so I merged both changes. Sorry if that wasn't clear!

Reply:
Ah, I didn't even notice that change from "can" to "need". Makes more sense now!